numpy/lib/stride_tricks.py

```diff
@@ -203,21 +203,36 @@ def sliding_window_view(x, shape, axis=None, *, subok=False, writeable=False):
 
     The axis can be specified explicitly:
 
-    >>> sliding_window_view(x, 3, 0)
+    >>> np.sliding_window_view(x, 3, 0)
     array([[[ 0, 10, 20],
             [ 1, 11, 21],
             [ 2, 12, 22],
             [ 3, 13, 23]]])
 
     The same axis can be used several times:
 
-    >>> sliding_window_view(x, (2, 3), (1, 1))
+    >>> np.sliding_window_view(x, (2, 3), (1, 1))
     array([[[[ 0,  1,  2],
              [ 1,  2,  3]]],
            [[[10, 11, 12],
              [11, 12, 13]]],
            [[[20, 21, 22],
              [21, 22, 23]]]])
+
+    Combining with stepped slicing (`::step`), this can be used to take sliding
+    views which skip elements:
+    >>> x = np.arange(7)
+    >>> np.sliding_window_view(x, 5)[:, ::2]
+    array([[0, 2, 4],
+           [1, 3, 5],
+           [2, 4, 6]])
+    or views which move by multiple elements
```

Comment:
I think this need blank lines
```suggestion
    Combining with stepped slicing (`::step`), this can be used to take sliding
    views which skip elements:

    >>> x = np.arange(7)
    >>> np.sliding_window_view(x, 5)[:, ::2]
    array([[0, 2, 4],
           [1, 3, 5],
           [2, 4, 6]])

    or views which move by multiple elements
```

Reply:
You are right.